FirebaseStorage.podspec

```diff
@@ -27,6 +27,8 @@ Firebase Storage provides robust, secure file uploads and downloads from Firebas
   s.tvos.deployment_target = tvos_deployment_target
   s.watchos.deployment_target = watchos_deployment_target
 
+  s.swift_version = '5.3'
```

Comment:
I'm not sure this is needed since there's no Swift in the pod. Or did this fix a warning with tests?

Reply:
Yes, it was to fix a warning that popped up during CI for this PR. See here: https://github.com/firebase/firebase-ios-sdk/runs/4610188684?check_suite_focus=true

The storage workflows get triggered very easily, my guess was the warning was caused by Storage depending on Core which now has swift?

Reply:
In that case, we might need to add it to all the podspecs that depend on FirebaseCore ...

Reply:
Good point. I guess that a podspec depending on a Swift pod means the swift_version must also be set to avoid the warning. 

Once this PR is sorted out. I could run touch the Gemfile to run CI for everything and see what happens.

Reply:
TODO